src/ol/ImageTile.js

```diff
@@ -8,8 +8,7 @@ import {listenOnce, unlistenByKey} from './events.js';
 import EventType from './events/EventType.js';
 
 /**
- * @typedef {function(new: ImageTile, import("./tilecoord.js").TileCoord,
- * TileState, string, ?string, import("./Tile.js").LoadFunction)} TileClass
+ * @typedef {typeof ImageTile} TileClass
```

Comment:
Would be even nicer to use `{typeof ImageTile}` directly where `{TileClass}` is currently used, and get rid of the `TileClass` typedef.

Reply:
I agree, updated to replace `TileClass`.